### PR TITLE
Add clickBlockToolbarButton end to end test util

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
This PR adds an end 2 end test util that allows clicking in a block toolbar button and refactors existing code to use it.

## How has this been tested?
We only need to verify that the end 2 end tests pass.

### CODE REVIEW

File: packages/e2e-test-utils/src/click-block-toolbar-button.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Clicks a block toolbar button
```
Comment (gziolo) [inline packages/e2e-test-utils/src/click-block-toolbar-button.js]:
```suggestion
 * Clicks a block toolbar button.
```

File: packages/e2e-tests/specs/blocks/list.test.js [hunk 2]
```diff
@@ -126,9 +127,7 @@ describe( 'List', () => {
 		await page.keyboard.press( 'Enter' );
 		// Pointer device is needed. Shift+Tab won't focus the toolbar.
```
Comment (gziolo) [inline packages/e2e-tests/specs/blocks/list.test.js]: Should those comments be removed? I don't quite understand why they are here without a broader context.

**Review by gziolo — APPROVED** [review]
I had some minor comments. Otherwise looks great. Thanks for adding this new helper. It simplifies code a bit 👍

**Comment by jorgefilipecosta** [comment]
Hi @gziolo thank you for the review, your comments were addressed before the merge.

**Pull request merged**